pakyow-test/lib/test_help/ext/request.rb

```diff
@@ -1,7 +1,9 @@
 module Pakyow
   class Request
-    def params
-      env['pakyow.params']
+    old_params = instance_method(:params)
+
+    define_method :params do
+      env.fetch('pakyow.params', old_params.bind(self).())
```

Comment:
Maybe use the block form of Hash#fetch to avoid calling `old_params` when not needed
```
env.fetch("pakyow.params") { old_params.bind(self).call }